ArduCopter/motors.cpp

```diff
@@ -176,10 +180,26 @@ void Copter::motors_output()
         motors->output();
     }
 
+    // implement motor kill for parachute testing
+    for (uint8_t i = 0; i < MOTOR_COUNT; ++i) {
+        if (motor_kill_all || motor_kill[i]) {
+            hal.rcout->write(i, 1000);
+        }
+    }
+
     // push all channels
     SRV_Channels::push();
 }
 
+void set_motor_kill(uint8_t motor_num, bool kill)
+{
+    if (motor_num == 0xFF) {
```

Comment:
Can the magic number be #define or const espr?

Reply:
Yes, just added a line `#define UINT_8BIT_MAX 255` and I now check `if (motor_num == UINT_8BIT_MAX)` instead of `0xFF`.